.config/spellcheck.toml

```diff
@@ -10,7 +10,12 @@ tokenization_splitchars = "\",;:.!?#(){}[]|/_-‒'`&@§¶…"
 extra_dictionaries      = ["forest.dic"]
 
 [hunspell.quirks]
-transform_regex     = []
+transform_regex     = [
+  # 10.7%
+  "^[0-9_]+(?:\\.[0-9]*)?(x|%)$",
+  # single char `=` `>` `%` ..
+  "^=|>|<|%$",
+]
```

Comment:
I'm not sure we should allow these characters. If we have equations in the documentation, they should be marked as code snippets.